### ISSUE TITLE
twisted.names.hosts.Resolver.lookupIPV6Address misdocuments its return type

### ISSUE THREAD

**Issue by twisted-trac** [twisted/twisted#6246]
|[<img alt="exarkun's avatar" src="https://avatars.githubusercontent.com/u/254565?s=50" width="50" height="50">](https://github.com/exarkun)| @exarkun reported|
|-|-|
|Trac ID|trac#6246|
|Type|defect|
|Created|2013-01-09 19:37:38Z|

It says `Record_A`, but it returns `Record_AAAA` instances.

Attachments:

* [my-twisted-patch.patch](https://raw.githubusercontent.com/twisted/twistedmatrix.com-trac-attachments/trunk/ticket/1a7/1a7f2740676e95717749fb462371f49df623162c/d5bf14bfe4ce8fc7b9256a069dbe396aeaef28e5.patch) (492 bytes) - added by esacteksab on 2013-01-15 04:09:03Z - 

<details><summary>Searchable metadata</summary>

```
trac-id__6246 6246
type__defect defect
reporter__exarkun exarkun
priority__normal normal
milestone__ 
branch__ 
branch_author__ 
status__closed closed
resolution__fixed fixed
component__names names
keywords__easy_documentation easy documentation
time__1357760258000000 1357760258000000
changetime__1358292289000000 1358292289000000
version__None None
owner__exarkun exarkun
cc__esacteksab
```
</details>

**Comment by twisted-trac** [comment]
|[<img alt="exarkun's avatar" src="https://avatars.githubusercontent.com/u/254565?s=50" width="50" height="50">](https://github.com/exarkun)<a name="note_3"></a>|@exarkun set owner to @exarkun|
|-|-|

Thanks!  This looks like the right fix - and better than I expected, even, since I overlooked the incorrect _IPv4_ earlier in the docstring.  The change needs a news fragment to go with it, but it's minor enough that I'll just create a `.misc`.  Also, the docstring should be rewrapped so as not to exceed 80 columns.  I'll go ahead and make those minor changes now and then apply the patch, thanks again.

**Comment by twisted-trac** [comment]
|[<img alt="exarkun's avatar" src="https://avatars.githubusercontent.com/u/254565?s=50" width="50" height="50">](https://github.com/exarkun)<a name="note_4"></a>|@exarkun set status to `closed`|
|-|-|

(In [36825]) Fix API documentation for twisted.names.hosts.Resolver.lookupIPV6Address

Author: esacteksab
Reviewer: exarkun
Fixes: [#6246](https://github.com/twisted/twisted/issues/6246)


Fix `lookupIPV6Address` API documentation to talk about IPv6 and AAAA records instead of
IPv4 and A records.
